box/resources/TestChainValidationMessageSuiteTestAccountActorCreationfailcreateBLSaccountactorinsufficientbalance

```diff
@@ -0,0 +1 @@
+0
```

Comment:
ask discussed afk, once this is rebased on master these files can all be removed since blob.go is human readable in latest change

Reply:
Done